Login

alsoza94 opened the pull request.

alsoza94 committed:
Conexion

Archivo conexion con la db.

alsoza94 committed:
Login

login para el ingreso al sistema

alsoza94 committed:
Css de login

Hoja de estilo del login

alsoza94 committed:
pagina que procesa el login

Se realiza todo el proceso de validacion de ingreso de nombre de usuario y contraseña

alsoza94 committed:
Rename procesaar_login.php to procesar_login.php

alsoza94 committed:
css

actualizacion del css

alsoza94 committed:
actualizacion procesar login

actualizacion de procesar login.php

alsoza94 committed:
procesar registro

pagina php para procesar el registro del usuario

alsoza94 committed:
pagina de registro

codigo html del registro

alsoza94 committed:
actualizacion procesar_login encri

Le agregue metodos de verificacion de contraseñas encriptadas

alsoza94 committed:
actualizacion procesar registro encri

Le agregue metodos de encriptacion al momento de registrar

alsoza94 committed:
procesar login

Actualizacion procesar_login.php

alsoza94 committed:
Login

Actualizacion login

alsoza94 committed:
Css

Actualizacion css

alsoza94 committed:
procesar_registro

Actualizacion procesar registro

alsoza94 committed:
sql base

SQL base

S4MU commented:
No hay un index.php
login_success.php no existe cuando el usuario ingresa al sistema
¿Por qué el nombre, apellidos y correo electrónico están encriptados en la table "persona"?
¿Por qué hay dos tablas? "persona" y "usuario"?

alsoza94 committed:
actualizacion login

Cambio nombre login a index

alsoza94 committed:
actualizacion login

alsoza94 committed:
login_sucess

Una vez verificado los datos del usuario, ingresa a esta pagina, solo para validar que el login fue exitoso

alsoza94 commented:
1-El login.php es el index, pero ya hice el cambio de nombre correspondiente.
2-El login_success.php es una pagina que me indica que realmente todo el proceso de login esta funcionando adecuadamente, dado que en lo solicitado solo se indico hacer un login.
3-Encripto los datos porque son datos sensibles, un tema de seguridad importante, no es recomendado ingresar a la base el nombre, apellidos, e inclusive (que no fue este el caso) la cedula asi sin encriptar, por lo menos que al momento de que si sucede un ataque, pues la informacion no sea entendible. Ahora, si el cliente no quiere los datos encriptados, pues se elimina ese mecanismo de seguridad.
4-Pues en el requerimiento del login no se indicaba que datos debian ser considerados al momento de hacer el registro del usuario, entonces considere el nombre y apellidos junto con el correo, ademas del nombre de usuario y contraseña, y estan en dos tablas separadas por temas de normalizacion. Igual, y como mencione anteriormente, si el cliente solo quiere que se registre un nombre de usuario y contraseña, pues edito el proceso de registro y elimino los demas datos que se requieren, o si quiere que todo se guarde en una sola tabla pues, igual edito los inserts y todo lo mando a la tabla usuario.

alsoza94 committed:
Actualizacion

actualizacion del proceso del login y registro
